docs/plugins.md

```diff
@@ -13,10 +13,10 @@ Let's take a quick look at the current dokku nginx plugin that's shipped with do
 ## Installing a plugin
 
 ```shell
-cd /var/lib/dokku/plugins/available
-git clone <git url>
-dokku plugin:install
+dokku plugin:install <git url>
 ```
+Note: `plugin:install` might trigger actions like `apt-get`, please make sure you inspect any potential plugin's `install` or `install-dependencies` trigger, and (of course) execute this command as the `root` user
+
```

Comment:
Basically I think this comment outside of the codeblock should be inside the codeblock and we should have ` sudo` prefix on the command. Something like this:

```shell
# This command requires `root` permissions as the `install` and `install-dependencies`
# plugin triggers may utilize commands such as `apt-get`. For non-core plugins, please
# inspect those plugins before running the following command as `root` user.
sudo plugin:install <git_url>

# previous versions (0.3.x and below) of dokku require a manual process to install plugins
cd /var/lib/dokku/plugins/available
sudo git clone <git url>
sudo dokku plugin:install
```

Does that make sense? Keeping the root notes outside of the codeblock is an easy way for users to get confused and complain that the docs are bad because they can't read ;)